tests/pytests/pkg/integration/test_version.py

```diff
@@ -5,26 +5,45 @@
 import pytest
 from pytestskipmarkers.utils import platform
 
+pytestmark = [
+    pytest.mark.skip_unless_on_linux,
+]
+
 
-@pytest.mark.skip_on_windows
 def test_salt_version(version, install_salt):
     """
     Test version output from salt --version
     """
+    actual = []
     test_bin = os.path.join(*install_salt.binary_paths["salt"])
```

Comment:
Maybe we should use `salt-call` here... then it will work on All OSes. No reason why we can't test this on Windows and MacOS. It's one of the few cross-platform tests we could run.

Reply:
The rest of the tests won't and didn't, hence the skip unless on Linux, see https://github.com/saltstack/salt/issues/66672, test_version.py is mentioned as needs fixing